tests/suit/utils/storagemap.go

```diff
@@ -47,6 +47,15 @@ func NewStorageMap(namespace string, providerIdentifier forkliftv1.Provider, sto
 		sdPairs = append(sdPairs, pair)
 	}
 
+	if providerIdentifier.Type() == api.OpenStack {
```

Comment:
why not setting it as part of the storage mapping that is created in openstack_test.go?

Reply:
I don't see that we manipulate it in the specific provider tests, OVA also has a specific branch above

Reply:
I mean to define the mapping of glance to the same target storage class [here](https://github.com/kubev2v/forklift/blob/main/tests/suit/openstack_test.go#L64) to reflect what clients are expected to provide

Reply:
I understood, what I meant is that we don't provider specific manipulations there